pkgs/by-name/ta/tachidesk-server/package.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, stdenvNoCC
+, fetchurl
+, makeWrapper
+, jdk17_headless
+, nixosTests
+}:
+
+let
+  jdk = jdk17_headless;
+in
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "tachidesk-server";
+  version = "0.7.0";
+  revision = 1197;
+
+  src = fetchurl {
+    url = "https://github.com/Suwayomi/Tachidesk-Server/releases/download/v0.7.0/Tachidesk-Server-v${finalAttrs.version}-r${toString finalAttrs.revision}.jar";
+    hash = "sha256-4DO1WiBCu/8ypFgJdBmEwQXQ1xaWAlbt8N5TELomVVA=";
+  };
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+
+  buildCommand = ''
+    makeWrapper ${jdk}/bin/java $out/bin/tachidesk-server --add-flags "-Dsuwayomi.tachidesk.config.server.initialOpenInBrowserEnabled=false -jar $src"
+  '';
+
+  passthru.tests = {
+    tachidesk-server = nixosTests.tachidesk-server;
+  };
+
+  meta = with lib; {
+    description = "A free and open source manga reader server that runs extensions built for Tachiyomi.";
+    longDescription = ''
+      A free and open source manga reader server that runs extensions built for Tachiyomi.
+
```

Comment:
```suggestion
```
we don't need to repeat the description again, as it is usually right next to each other anyway.

Reply:
Done